src/workerd/api/sql-test.js

```diff
@@ -1251,6 +1256,11 @@ export default {
 
     // Everything's still consistent.
     assert.equal(await doReq('increment'), 3);
+
+    // Delete all: increments start over
+    await doReq('deleteAll');
+    assert.equal(await doReq('increment'), 1);
+    assert.equal(await doReq('increment'), 2);
```

Comment:
nit: a variation on this that also waits on these concurrently would be nice.

```js
  const results = await Promise.all([
    doReq('deleteAll'),
    doReq('increment'),
    doReq('increment'),
  ]);
  assert.deepStrictEqual(results, ...);
```

Largely to make sure that operations are correctly sequenced.

Reply:
I don't understand... what would that test that is interesting here?